cmd/mirror-main.go

```diff
@@ -92,10 +95,16 @@ var (
 			Name:  "disable-multipart",
 			Usage: "disable multipart upload feature",
 		},
-		cli.StringSliceFlag{
-			Name:  "exclude",
-			Usage: "exclude object(s) that match specified object name pattern",
-		},
+		newExcludeWildcardFilterFlag(
+			"exclude",
+			"exclude object(s) that match specified object name pattern",
+			&nameFiltersValue,
+		),
+		newIncludeWildcardFilterFlag(
+			"include",
+			"include object(s) that match specified object name pattern",
+			&nameFiltersValue,
+		),
```

Comment:
The order in the code does not matter. Both flags share a single slice. Both flags work in order on the command line as the "AND" operator.  The processing cycle is located [here](https://github.com/minio/mc/pull/3865/commits/0a155fabab59fedbac7f073d1c73fad9f784d3bc#diff-cea9c9c27b238e950c3ecc79ea095a673897839acb94fc46155d3b271a50df51R44):

We can specify both flags multiple times, and each file will be processed by each flag.  [aws s3](https://docs.aws.amazon.com/cli/latest/reference/s3/#use-of-exclude-and-include-filters)  works in the same manner. 

The description of the flags is not so accurate. Ideally we should write a more descriptive text, but I need some help for that.

More examples is located in [the test file](https://github.com/minio/mc/pull/3865/commits/0a155fabab59fedbac7f073d1c73fad9f784d3bc#diff-469b28660f7edba08b73508db2975fba9c6e254cd36cd8aa8ed2f5ba30faa96cR53)

Reply:
@scalm I am thinking when conflicting/overlapping information is provided as `--include` and `--exclude` arguments.

Is is dependent on the parameter order on the commandline? If so, that should be documented, since that will probably be unexpected for many users.

Reply:
That's right, it depends on their order. Let's try to improve the descriptions.